spyder/config/main.py

```diff
@@ -437,7 +437,8 @@
               'editor/close file 1': "Ctrl+W",
               'editor/close file 2': "Ctrl+F4",
               'editor/run cell': CTRL + '+Return',
-              'editor/run cell and advance': 'Shift+Return',
+              'editor/run cell and advance': (
+                  CTRL + '+Shift+Return' if MAC else 'Shift+Return'),

```

Comment:
@dalthviz, we can't do this because it would change the way we run cells in Spyder, which (I think) many people would find unpleasant.

So, if the test you introduced is failing only in macOS, please skip it there.

Reply:
Ok :+1: but the reason I change its because seems like that shortcut collides with something else (don't know with what other shortcut) and even when you try to reapply it over the same action on MacOS.

Seems like if you try to change any action shortcut to `Shift+Return` it doesn't work although other shortcut changes work as expected, is that valid?

Reply:
Ok, that seems really serious. Thanks @dalthviz for finding it!

@steff456, could you confirm if `Shift+Enter` can't be remapped to perform other actions in macOS?

Reply:
Note: Checking a little bit seems that the ambiguous warning only appears when remapping `Shift+Return` in the editor